onnxruntime/core/optimizer/unsqueeze_elimination.h

```diff
@@ -3,16 +3,23 @@
 
 #pragma once
 
-#include "core/optimizer/graph_transformer.h"
+#include "core/optimizer/rewrite_rule.h"
 
 namespace onnxruntime {
 
-class UnsqueezeElimination : public onnxruntime::GraphTransformer {
+class UnsqueezeElimination : public RewriteRule {
  public:
-  UnsqueezeElimination() noexcept : onnxruntime::GraphTransformer("EliminateUnsqueeze", "Eliminate unsqueeze node") {}
+  UnsqueezeElimination() noexcept : RewriteRule("EliminateUnsqueeze", "Eliminate unsqueeze node") {}
 
  private:
-  Status ApplyImpl(onnxruntime::Graph& graph, bool& modified, int graph_level) const override;
+  /** Apply rule when op type is one of the following. */
```

Comment:
> one of the [](start = 32, length = 11)

nit: why one of the? are you planning to support a ";" delimiter based sequence?

Reply:
At some point I had it as a vector, so that people can add more (along the lines of the excluded ops in constant folding). But it does not make sense here, so I will fix it.